cypress-integration.config.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-console */
+const { defineConfig } = require("cypress");
+const fs = require("fs");
+
+module.exports = defineConfig({
+  projectId: "51ef7c",
+  chromeWebSecurity: false,
+  defaultCommandTimeout: 20000,
+  requestTimeout: 20000,
+  viewportWidth: 1400,
+  viewportHeight: 660,
+  retries: {
+    runMode: 2,
+    openMode: 1,
+  },
+  e2e: {
+    env: {
+      grepFilterSpecs: true,
+    },
+    setupNodeEvents(on, config) {
+      config = require("./cypress/support/cypress-grep/plugin")(config);
+      config = require("./cypress/plugins/index.js")(on, config);
+      on("after:spec", (spec, results) => {
+        if (results && results.video) {
+          return fs.unlink(results.video, function(err) {
+            if (err) {
+              console.warn(`Could not remove video - ${err}`);
+            } else {
+              console.log("File removed:", results.video);
+            }
+          });
+        }
+      });
+      return config;
+    },
+    baseUrl: "http://localhost:4173/",
+    specPattern: "cypress/functional/**/*.{js,jsx,ts,tsx}",
+  },
+});
```

Comment:
Do we need to have two config files, there are almost same. Wouldn't be easier to just add environment variable functional with default false, and base on that create spec pattern? It could be run then via "npx cypress open --env functional=true"

Reply:
Please don't create 2 config files if it is not a must.